stdlib/mmap.pyi

```diff
@@ -16,6 +16,8 @@ if sys.platform == "linux":
     MAP_EXECUTABLE: int
     if sys.version_info >= (3, 10):
         MAP_POPULATE: int
+    if sys.version_info >= (3, 11):
+        MAP_STACK: int
```

Comment:
The comment in the CPython source code seems to indicate that these also exist on some more exotic platforms -- we should probably do our best to account for those, even if we don't test for them in CI

```suggestion
if sys.version_info >= (3, 11) and sys.platform != "win32" and sys.platform != "darwin":
    MAP_STACK: int
```